crates/starknet/src/starknet/mod.rs

```diff
@@ -580,19 +621,24 @@ mod tests {
     #[test]
     fn getting_state_reader_at_block_by_hash() {
```

Comment:
I see I forgot to rename the tests when I renamed the function from `get_state_reader_at` to `get_state_at`. Also, the name of the test could reflect the fact that it's an unhappy path (or that it should fail): `getting_state_reader_at_block_by_nonexistent_hash`

Reply:
[8757a47](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/104/commits/8757a47ecd6e798fc5c85f4251a68aa8734087c0)